php-checks/src/test/java/org/sonar/php/checks/StringLiteralDuplicatedCheckTest.java

```diff
@@ -19,28 +19,22 @@
  */
 package org.sonar.php.checks;
 
-import com.google.common.collect.ImmutableList;
 import org.junit.Test;
-import org.sonar.php.tree.visitors.LegacyIssue;
-import org.sonar.plugins.php.TestUtils;
-import org.sonar.plugins.php.api.tests.PHPCheckTest;
-import org.sonar.plugins.php.api.visitors.PhpIssue;
-
-import java.util.List;
+import org.sonar.plugins.php.CheckVerifier;
 
 public class StringLiteralDuplicatedCheckTest {
 
   private StringLiteralDuplicatedCheck check = new StringLiteralDuplicatedCheck();
 
   @Test
   public void defaultValue() throws Exception {
-    PHPCheckTest.check(check, TestUtils.getCheckFile("StringLiteralDuplicatedCheck.php"));
+    CheckVerifier.verify(check, "StringLiteralDuplicatedCheck/default.php");
   }
 
   @Test
   public void custom() throws Exception {
     check.threshold = 4;
-    List<PhpIssue> issue = ImmutableList.<PhpIssue>of(new LegacyIssue(check, "Define a constant instead of duplicating this literal \"name1\" 4 times.").line(20));
-    PHPCheckTest.check(check, TestUtils.getCheckFile("StringLiteralDuplicatedCheck.php"), issue);
+    check.minimalLiteralLength = 4;
```

Comment:
If `threshold` and `minimalLiteralLength` have the same value, how can we be sure that we handle those properties correctly? If those properties are inverted by mistake, this test will not catch it.